Sources/Hyperspace/Request/NetworkRequest.swift

```diff
@@ -113,6 +113,18 @@ public extension NetworkRequest {
         
         return request
     }
+    
+    func modifyingHeaders(_ headers: [HTTP.HeaderKey: HTTP.HeaderValue]?) -> Self {
+        var copy = self
+        copy.headers = headers
+        return copy
+    }
+    
+    func modifyingBody(_ body: Data?) -> Self {
```

Comment:
I'm assuming we'll want to use whatever naming pattern we decide on here too.

Reply:
Yea. Will switch over to `usingHeaders`, `replacingHeaders(with:)` seems a little to verbose to me, and conveys the same thing.